_data/harishhari0525/harishhari0525.md

```diff
@@ -0,0 +1,7 @@
+-----
```

Comment:
Please remove an extra dash from here,
It should look something like this `---`